packages/components/src/navigation/index.js

```diff
@@ -67,6 +67,14 @@ const Navigation = ( { activeItemId, children, data, rootTitle } ) => {
 		}
 	}, [ activeItem ] );
 
+	const isMounted = useRef( false );
```

Comment:
I think this doesn't need an initial value, e.g. `const isMounted = useRef();` should work just as well.